playerUi/timelineMarkersWithImages.html

```diff
@@ -0,0 +1,163 @@
+      width: 25%;
+      min-width: 350px;
+      float: left;
+      margin: auto;
+    }
+
+    .title {
+      width: 75%;
+      white-space: nowrap;
+    }
+
+    .clear {
+      clear: both;
+    }
+
+    .content {
+      margin-bottom: 10em;
+    }
+
+    h1,
+    h2,
+    h3,
+    p {
+      font-weight: 300;
+      text-align: center;
+      margin: 40px;
+    }
+
+    #player {
+      max-width: 900px;
+      width: 90%;
+      margin: auto;
+      -webkit-box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+      -moz-box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+      box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+    }
+
+    a {
+      color: #97d9ef;
+      font-weight: 400;
+      text-decoration: none;
+    }
+
+    a:hover {
+      color: #fff;
+    }
+
+    @media (max-width: 800px) {
+      .logo {
+        width: 100%;
+      }
+
+      .title {
+        display: none;
+      }
+    }
+  </style>
+</head>
+
+<body>
+  <div id="wrapper">
+    <div id="banner">
+      <div class="logo"><img src="../images/bitmovin-logo.png"></div>
+      <div class="title">
+        <h1>Timeline markers</h1>
+      </div>
+      <div class="clear"></div>
+    </div>
+    <div class="container">
+      <h1>HTML5 Adaptive Streaming Player for MPEG-DASH & HLS</h1>
+      <h2>Your videos play everywhere with low startup delay, no buffering and in highest quality.</h2>
+      <div class="content">
+        <div class="player-wrapper">
+          <div id="player"></div>
+        </div>
+        <div class="description">
+          <p>For more information about the bitmovin player, please have a look at our online <a
+              href="//bitmovin.com/docs" target="_blank">Documentation</a>.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <script type="text/javascript">
+    var conf = {
+      key: "YOUR-PLAYER-LICENSE-KEY",
+    };
+
+    var imageUrl = './../images/bit-fav.png';
+  
+    var sourceConfig = {
+      title: "Bitmovin Player",
+      description: "This demo showcases the possibility to define markers in the timeline with images",
+      dash: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd",
+      hls: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8",
+      progressive: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/MI201109210084_mpeg-4_hd_high_1080p25_10mbits.mp4",
+      markers: [
+        { time: 10, title: 'Marker 1', imageUrl },
+        { time: 65, duration: 1, title: 'Marker 2', imageUrl },
+        { time: 105, duration: 10, title: 'Marker 3', imageUrl },
+      ]
+    };
+
+    var player = new bitmovin.player.Player(document.getElementById('player'), conf);
+
+    player.load(sourceConfig).then(function () {
+      console.log('Successfully loaded source');
+    }, function () {
+      console.log('Error while loading source');
+    });
+  </script>
+</body>
+
+</html>
```

Comment:
Here as well 🙂

Reply:
Addressed in https://github.com/bitmovin/bitmovin-player-web-samples/pull/46/commits/db03c723ea8342eaea3139a26346ba17708568c9